src/app/cesium.service.ts

```diff
@@ -509,6 +514,58 @@ export class CesiumService {
   }
 
 
+  public getAndLoadTfrNoFlyZones(): void {
+    console.log('Loading TFR No FlyZones');
+
+    this.httpClient
+      .get<GetNoFlyZonesResponse>(
+        Url.consumer('/getTfrNoFlyZones'),
+        this.httpOptions
+      )
+      .subscribe((data) => {
+        this.getTfrNoFlyZoneResponse = data;
+
+        console.log('TFR NO FLY ZONES')
+        console.log(this.getTfrNoFlyZoneResponse);
+        for (const tfrNoFly of this.getTfrNoFlyZoneResponse.tfrNoFlyZones) {
+          
+          if(tfrNoFly.notamType === "BOUNDARY") {
+            console.log('Boundary')
+            this.global_viewer.entities.add( {
+              polygon: {
+                hierarchy: Cesium.Cartesian3.fromDegreesArray(tfrNoFly.latlong),
+                height: 0,
+                material: Cesium.Color.RED.withAlpha(0.5),
+                outline: true,
+                outlineColor: Cesium.Color.BLACK,
+              },
+            }
+            );
+          }
+
+          if(tfrNoFly.notamType === "RADIUS") {
+            this.global_viewer.entities.add({
```

Comment:
In the future, we would want these objects to be managed. They should be typed as static Typescript type and tracked within another class that should control how they are rendered and when they should be unmounted.